runtime/onert/core/src/ir/operation/BCQFullyConnected.cc

```diff
@@ -31,7 +31,7 @@ void BCQFullyConnected::accept(OperationVisitor &v) const { v.visit(*this); }
 
 BCQFullyConnected::BCQFullyConnected(const OperandIndexSequence &inputs,
                                      const OperandIndexSequence &outputs, const Param &param)
-    : Operation{OperandConstraint::createInRange(4u, 5u), inputs, outputs}, _param{param}
+    : Operation{OperandConstraint::createExact(5u), inputs, outputs}, _param{param}
```

Comment:
I once put this in cerateInRange(4u, 5u) because bias is optional, but FullyConnected itself has createdExact(3u). @glistening I just followed FullyConnected's implementation but I wonder how to set this OperandConstraint in case of optional tensor.

Reply:
@glistening 
Following to our offline conversation,
- Optional tensor is currently regarded as -1 tensor in runtime, So it is counted as OprandConstrants. Therefore createExact(5u) is alright. I will leave it like this.